.env.example

```diff
@@ -1,45 +1,22 @@
 CELERY_TASK_ALWAYS_EAGER=True
 DJANGO_LOG_LEVEL=INFO
 LOG_LEVEL=INFO
+OPEN_DISCUSSIONS_DEFAULT_SITE_KEY=
+OPEN_DISCUSSIONS_BASE_URL=http://od.odl.local:8063
```

Comment:
elsewhere in the docs it says to set this to `OPEN_DISCUSSIONS_BASE_URL=http://docker.for.mac.localhost:8063/` (On Mac)

This settings here seems correct to me. Maybe the other one should be changed?

Reply:
It's mentioned currently at https://github.com/mitodl/open-discussions/blob/master/README.md#L175 in the section https://github.com/mitodl/open-discussions/blob/master/README.md#6-integration-with-micromasters

Reply:
@pdpinch The `docker.for.mac.localhost` value is only set on `MICROMASTERS_BASE_URL`, not `OPEN_DISCUSSIONS_BASE_URL`. Those two values are and should be different. I have `OPEN_DISCUSSIONS_BASE_URL=http://od.odl.local:8063` and `MICROMASTERS_BASE_URL=http://docker.for.mac.localhost:8079/` in my own `.env` file and it works